spec/models/spree/payment_spec.rb

```diff
@@ -227,7 +227,7 @@
 
           context "if successful" do
             before do
-              payment.payment_method.should_receive(:capture).with(payment, card, anything).and_return(success_response)
+              payment.payment_method.should_receive(:capture).and_return(success_response)
```

Comment:
it'd be great if we could refactor it to use RSpec 3's `allow().to` syntax while we are at it.

Reply:
👍 694151f9c7b17e2da0512c009ad92a6a04b7721d